code/concepts/README.md

```diff
@@ -0,0 +1,18 @@
+# Concepts
+
+Here we explore:
+- how <=> are == differing.
```

Comment:
```suggestion
- how <=> and == differ.
```

Reply:
To be replaced, actually. A copy-paste waste.